libs/test/SalesforceSDKTest/src/com/salesforce/androidsdk/app/SalesforceSDKManagerTest.java

```diff
@@ -101,15 +102,45 @@ public void testDefaultAiltnAppName() {
      */
     @Test
     public void testOverrideInvalidAiltnAppName() {
-        SalesforceSDKTestManager.init(InstrumentationRegistry.getInstrumentation().getTargetContext(), MainActivity.class);
+        SalesforceSDKTestManager.init(getInstrumentation().getTargetContext(), MainActivity.class);
         SalesforceSDKTestManager.setAiltnAppName(null);
         SalesforceSDKTestManager.getInstance().setIsTestRun(true);
         compareAiltnAppNames(getDefaultAppName());
     }
 
+    /**
+     * Test the default theme value.
+     */
+    @Test
+    public void testDefaultTheme() {
+        SalesforceSDKTestManager.init(getInstrumentation().getTargetContext(), MainActivity.class);
+        Assert.assertFalse(SalesforceSDKTestManager.getInstance().isDarkTheme());
```

Comment:
If I run that test on a device that uses dark theme, it will fail.

Reply:
Good catch, the default for a new device might even change in the future.  I have updated the test.